src/xdocs/config_misc.xml

```diff
@@ -2078,6 +2078,55 @@ d = e / f;        // Another comment for this line
   &lt;property name=&quot;format&quot; value=&quot;^\\s*$&quot;/&gt;
 &lt;/module&gt;
         </source>
+
+        <p>
+          To allow certain trailing comments, violations should be suppressed.
+          Example:
+        </p>
+
+        <source>
+public class Test {
+  int a; // SUPPRESS CHECKSTYLE, OK
+  int b; // NOPMD, OK
+  int c; // NOSONAR, OK
+  int d; // violation, not suppressed
+}
+        </source>
+
+        <p>
+          To configure check so that trailing comment with exact comments like
+          &quot;SUPPRESS CHECKSTYLE, OK&quot;, &quot;NOPMD, OK&quot;, &quot;NOSONAR, OK&quot;
+          are not shown as violations:
+        </p>
+
+        <source>
+&lt;module name=&quot;TrailingComment&quot;/&gt;
+&lt;module name=&quot;SuppressionXpathSingleFilter&quot;&gt;
```

Comment:
your example is more about `SuppressionXpathSingleFilter ` than `TrailingComment `, so I would recommend to move examples there

Reply:
Sir, in the issue, it was that mentioned that we need to show users that it is better to use this check with a bunch of satellite filters, are they are shown here

Reply:
https://github.com/checkstyle/checkstyle/pull/9832#discussion_r630703195
Agree, filters make more sense. Users cannot suppress `TrailingComment` through it's own module. It needs the xpath filters to do a suppression.

Reply:
Done. @timurt Please restart the review process.